src/store/messagesStore.js

```diff
@@ -35,6 +35,7 @@ import { showError } from '@nextcloud/dialogs'
 import {
 	ATTENDEE,
 } from '../constants'
+import { cloneDeep } from 'lodash'
```

Comment:
please import with `import cloneDeep from 'lodash/cloneDeep'` of whatever works.

last time we used this kind of import there were weird unexplained side effects in the files app. Ref: https://github.com/nextcloud/spreed/issues/5858